### DISCUSSION TITLE
Tutorial: explain used CSS attributes

### DISCUSSION THREAD

**Discussion by michcio1234** [Textualize/textual#2431] [Q&A]
Please explain CSS attributes used in [Dynamic CSS](https://textual.textualize.io/tutorial/#dynamic-css) section of the tutorial, namely `display: block` and `visibility: hidden` (and its difference to `display: none`).

**Comment by davep** [comment]
You can find all the CSS styles documented [in the reference section of the docs](https://textual.textualize.io/styles/). See [`display`](https://textual.textualize.io/styles/display/) and [`visibility`](https://textual.textualize.io/styles/visibility/).
